TypeCobol.Test/Parser/Programs/Cobol85/SubtractionAsUserDefinedWord.rdzMix.txt

```diff
@@ -0,0 +1,152 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. DVZF0OSM.
+       data division.
+       working-storage section.
+        01 a pic 9.
+        01 300-200X  pic 9 value 1.
+        01 200X-100X pic 9 value 1.
+      *KO variable must contains at least one alphabetic char
+Line 9[12,18] <27, Error, Syntax> - Syntax error : Variable name must contain at least one alphabetic char.
+        01 400-500 pic 9.
+
+       procedure division.
+      *Should be KO. 400-500 is not a valid variable name because it
+      *   contains no alphabetic char
+      *   The parser will only report the error on the variable 
+      *   declaration.
+           compute a = 400-500
```

Comment:
If it's currently ok for the parser, does it mean that it builds a compute statement and '400-500' becomes an arithmetic operation ?

Reply:
No, it means `400-500` will be seen as a StorageArea.